test/main/main_spec.rb

```diff
@@ -0,0 +1,101 @@
+# These tests start a real server and talk to it over TCP so we can
+# isolate the effects of require "angelo/main" in a separate process
+# but still test them.
+#
+# See test/main/app.rb for the code of the app we test against.
+
+require_relative '../spec_helper'
+
+class MainSpec < Minitest::Spec
+
+  # Info for starting the server.
+
+  app_file = File.expand_path("../app.rb", __FILE__)
+  address = "127.14.4.10"
```

Comment:
Bummer.  It's an address on the localhost network.  I guess it doesn't work
on all OSes.  I just stuck it there to avoid port collisions which are
unlikely anyway.  I did that because running the angelo tests often breaks
on my system because I've often got something running on the same port
which is a PITA.

127.0.0.1 is fine.

On Thu, Jan 29, 2015 at 11:18 AM, kenichi nakamura <notifications@github.com
> wrote:

> In test/main/main_spec.rb
> <https://github.com/kenichi/angelo/pull/37#discussion_r23794290>:
>
> > @@ -0,0 +1,101 @@
> > +# These tests start a real server and talk to it over TCP so we can
> > +# isolate the effects of require "angelo/main" in a separate process
> > +# but still test them.
> > +#
> > +# See test/main/app.rb for the code of the app we test against.
> > +
> > +require_relative '../spec_helper'
> > +
> > +class MainSpec < Minitest::Spec
> > +
> > +  # Info for starting the server.
> > +
> > +  app_file = File.expand_path("../app.rb", __FILE__)
> > +  address = "127.14.4.10"
>
> i had to change this to 127.0.0.1 to get tests working, what is this
> address?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kenichi/angelo/pull/37/files#r23794290>.
>